proto/one/historian/ingest/file_configuration.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+import "google/protobuf/wrappers.proto";
+
+option csharp_namespace ="ONE.Models.CSharp";
+
+message FileConfiguration
+{
```

Comment:
please follow to convention of { being on the same line as the message declaration